tests/product/test_server_install.py

```diff
@@ -335,10 +336,11 @@ def test_install_interactive_with_ips(self):
         self.copy_presto_rpm_to_master()
 
         cmd_output = self.run_prestoadmin_script(
-            "echo -e 'root\n22\n%s\n%s\n' | "
-            "./presto-admin server install /mnt/presto-admin/%s " %
+            'echo -e \'root\n22\n%s\n%s\n\' | '
```

Comment:
eh, I can change it back. I was using single quotes no matter what, but I don't feel that strongly about it.